py_ecc/fields/optimized_field_elements.py

```diff
@@ -428,6 +426,23 @@ def __init__(self, coeffs: Sequence[IntOrFQ]) -> None:
         self.mc_tuples = [(i, c) for i, c in enumerate(self.FQ2_MODULUS_COEFFS) if c]
         super().__init__(coeffs, self.FQ2_MODULUS_COEFFS)
 
+    @property
```

Comment:
Depending on how often these get accessed it might be worth leveraging `cached_property` for this and any other computed properties.